### DISCUSSION TITLE
Guidance on getting existing non-IaC environments into a well-architected/RefArch state

### DISCUSSION THREAD

**Discussion by iantybo** [gruntwork-io/knowledge-base#483] [Help] [answered]
Does Gruntwork have guidance on getting my environments which were not done entirely as IaC into a well-architected environment? Do the Reference Architecture and infrastructure-live repo design help this in any way? 

---

<ins datetime="2022-06-22T20:03:45Z">
  <p><a href="https://support.gruntwork.io/hc/requests/108819">Tracked in ticket #108819</a></p>
</ins>

**Comment by yorinasub17** [comment, marked as answer]
We currently do not support importing existing infrastructure into IaC. We recommend deploying new accounts from scratch using the Reference Architecture and migrating your workloads on to the new platform, rather than adapting existing AWS Accounts.
